### PR TITLE
Made redeem ticket calls more fire and forget

### PR DESCRIPTION
**Pull request by bryanjos**
connects to #718

### CODE REVIEW

File: src/state/eventManagerStateProvider.js [hunk 1]
```diff
@@ -61,6 +61,17 @@ export class EventManagerContainer extends Container {
     await this.setState({isFetchingGuests: false})
   }
 
+  updateGuestStatus = (guestId, newStatus) => {
+    const guests = this.state.guests.slice(0)
+
+    for (let i = 0; i < guests.length; i++) {
+      if (guests[i].id === guestId) {
```
Comment (daybreaker) [inline src/state/eventManagerStateProvider.js]: There's not a more efficient way to do this?

Comment (bryanjos) [reply src/state/eventManagerStateProvider.js]: More efficient than a loop? I don't think so. Not without knowing the index of the guest

**Comment by bryanjos** [comment]
Here it will refresh the list anyways, so if there is an error, the list will be updated
https://github.com/revelrylabs/bn-mobile-react/blob/deployment/src/Accounts/guest-list-checkin.js#L252-L264

**Comment by daybreaker** [comment]
But wouldnt it update the list, while the user who's ticket scan failed would have already gone into the venue?

**Comment by bryanjos** [comment]
That's a possibility. I don't know how to handle this besides using an optimistic approach. We can hold on merging until someone answers.

**Comment by bryanjos** [comment]
@daybreaker actually, I'll make it something like "processing" and then there shouldn't be any confusion

**Comment by zmelamed** [comment]
Just trying to make sure we're clear on what the problem is. Are we certain that it's related to the API response time? @Krakaw do you have any insight? 

Not sure what the list size is that you're testing with, but for an event with 500-10,000 list items on an older device could this looping method mentioned above impact speed significantly?

**Comment by Krakaw** [comment]
I'm running some tests to check the response times purely from the API across development, staging and production. I'll respond here with the results as soon as they are complete.

**Pull request merged**